packages/compass-components/src/components/form-modal.tsx

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { css } from '@leafygreen-ui/emotion';
+import { Variant as ButtonVariant } from '@leafygreen-ui/button';
+import { Modal } from './modal';
+import { Button, ModalFooter } from './leafygreen';
+import { Theme, withTheme } from '../hooks/use-theme';
+
+import { ModalContent } from './modal-content';
+import { ModalHeader } from './modal-header';
+
+export const Variant = {
+  Default: ButtonVariant.Primary,
+  Danger: ButtonVariant.Danger,
+} as const;
+
+export type Variant = typeof Variant[keyof typeof Variant];
+
+const baseModalStyle = css`
```

Comment:
Lots of CSS throughout that I copy/pasted from ConfirmationModal and will hopefully be able to clean up and get up to our standards.